src/litdata/streaming/writer.py

```diff
@@ -238,6 +245,11 @@ def _create_chunk(self, filename: str, on_done: bool = False) -> bytes:
 
         current_chunk_bytes = sum([item.bytes for item in items])
 
+        # Whether to encrypt the data at the chunk level
+        if self._encryption and self._encryption.level == "chunk":
```

Comment:
```suggestion
        if self._encryption and self._encryption.level == "chunk":
```

Can we replace all those checks with enum check ? 

```
self._encryption and self._encryption.level == EntrycryptionType.CHUNK
```